### PR TITLE
Videopage Corrected Issues

### PR DESCRIPTION
**Pull request by abouhid**
This pull request is being made with the execution of the final changes after the first code review.

### CODE REVIEW

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨
## Impressive project team 👏
However, we still need to make some changes before we can move on. Check the comments below.

File: index.html [hunk 1]
```diff
@@ -1,25 +1,345 @@
...
+          <button class="log-in">
+            <span> <i id="icon-profile" class="fa fa-user-circle"></i> </span
+            >SIGN IN
+          </button>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <div class="main"></div>
```
Comment (tirthajyoti-ghosh) [inline index.html]: ### This is an empty element. Please remove it.

File: index.html [hunk 2]
```diff
@@ -1,25 +1,345 @@
...
+
+      <div class="primary-column">
```
Comment (tirthajyoti-ghosh) [inline index.html:49]: ### The width of this section needs to be increased as pointed out by the previous reviewer.

File: index.html [hunk 3]
```diff
@@ -1,25 +1,345 @@
...
+        </div>
+
+        <div class="right-header">
+          <i id="icon-video" class="fas fa-video" style="color: #606060;"></i>
+          <i id="icon-grid" class="fas fa-th" style="color: #606060;"></i>
+          <i
+            id="icon-setting"
+            class="fa fa-ellipsis-v"
+            style="color: #606060;"
+          ></i>
```
Comment (tirthajyoti-ghosh) [inline index.html]: ### There should be no styles inside HTML files. Inline styling is a CSS bad practice. Please remove them.

File: README.md [hunk 4]
```diff
@@ -1 +1,37 @@
...
+
+Additional description about the project and its features:
+
+## Built With
+
+- Html and CSS.
+
+## Live Demo
+
+[live demo link](https://abouhid.github.io/youtube_project/)
```
Comment (tirthajyoti-ghosh) [inline README.md:15]: ### Live demo link is not updated with the applied changes from the previous reviewer.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
[OPTIONAL] The text in the like/dislike/share button section is sticking to the icons. A little bit of space between them would be good.

*I noticed you have opened a new PR for this review. It's not a good practice to open different PRs for different reviews. Next time use the same PR for all your reviews*

Apply these changes and request for another review.
Happy coding 👨‍💻

**Comment by karthy1998** [comment]
We have Corrected The issues that you have pointed out

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Great job implementing those changes 👏🙌 and your project meets all the requirements but one.

- Your HTML file still uses in-line CSS, which should be removed before we can merge. Check the comments below.

File: index.html [hunk 5]
```diff
@@ -1,25 +1,338 @@
...
+          <div class="com-block">
+            <div class="com-unity">
+              <span
+                ><img
+                  class="com-img"
+                  alt="video"
+                  src="https://lh3.googleusercontent.com/a-/AOh14Gg3z157SKsDNkciDW-e7qUfZIM01WXORpWQp8ShQA=s88"
+              /></span>
+
+              <div style="margin-left: 20px;">
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove in-line CSS. It's a bad practice -> ([article](https://speckyboy.com/good-bad-css-practices)).

File: index.html [hunk 6]
```diff
@@ -1,25 +1,338 @@
...
+              <span
+                ><img
+                  class="com-img"
+                  alt="video"
+                  src="https://avatars3.githubusercontent.com/u/60702127?s=400&u=45cf9e61814be275f703381092a3d2a3936ebbd8&v=4"
+              /></span>
 
-    </div>
-</div>
+              <div style="margin-left: 20px;">
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove in-line CSS.

File: index.html [hunk 7]
```diff
@@ -1,25 +1,338 @@
...
+            </div>
+            <div class="com-unity">
+              <span
+                ><img
+                  class="com-img"
+                  alt="video"
+                  src="https://yt3.ggpht.com/a/AATXAJzv_SRfKIDVZfoj2ji2nPLMTd-jNJsz2jNTRg=s48-c-k-c0xffffffff-no-rj-mo"
+              /></span>
 
+              <div style="margin-left: 20px;">
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove in-line CSS.

File: index.html [hunk 8]
```diff
@@ -1,25 +1,338 @@
...
+              <span
+                ><img
+                  class="com-img"
+                  alt="video"
+                  src="https://pbs.twimg.com/profile_images/1161128533986381824/khmFawef_400x400.jpg"
+              /></span>
 
-</body>
\ No newline at end of file
+              <div style="margin-left: 20px;">
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove in-line CSS.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Remove those inline styles and request another review.

Happy coding 👨‍💻

**Comment by abouhid** [comment]
We are sorry for this inline CSS, we had corrected it before but seems like we did not commit correctly.
Here is the final step of the project.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks:
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job @abouhid & @karthykarthick! :+1:
- You can now merge to master. I wish you all the very best in your next tasks :first_place_medal:
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**